### PR TITLE
#1: Adds docker and lua language servers

### LINKED ISSUE #1: Add existing language servers
This will add the `docker-langserver` and `lua-language-server` dockerfiles.

### PR DESCRIPTION
**Pull request by erkrnt**
Closes #1

### CODE REVIEW

File: docker-langserver/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.13.5 as build
+
+RUN apk add --no-cache nodejs npm
+
+RUN npm install -g dockerfile-language-server-nodejs
+
+WORKDIR /code
```
Comment (nogweii) [inline docker-langserver/Dockerfile:7]: this is a different `WORKDIR`, do we want to standardize these?

File: lua-language-server/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+FROM alpine:3.13.5 as build
+
+ENV VERSION="1.20.5"
```
Comment (nogweii) [inline lua-language-server/Dockerfile:3]: I like that the VERSION is specified here -- it'd be very cool to copy this as a pattern.

**Comment by darthmeme** [comment]
Looks awesome! Ship it! 🚀

File: docker-langserver/Dockerfile (hunk 1 above)
Comment (erkrnt) [reply docker-langserver/Dockerfile:7]: So I think this should be standardized but isn't required here - will remove.

**Comment by nogweii** [comment]
Actually, do we want to worry about running these as non-root users?

File: lua-language-server/Dockerfile (hunk 2 above)
Comment (erkrnt) [reply lua-language-server/Dockerfile:3]: I actually had this in the `docker-langserver` and removed it to debug - will re-add.

**Comment by erkrnt** [comment]
> Actually, do we want to worry about running these as non-root users?

Since these are local development containers and not actually being run in a `production` environment I think this is fine for now - but you're welcome to open a PR and make the changes.

**Review by nogweii — APPROVED** [review]
LGTM. :shipit:

**Pull request merged**